src/main.ts

```diff
@@ -26,6 +26,12 @@ async function run(): Promise<void> {
       process.chdir(inputs.workdir);
     }
 
+    if (inputs.projectUpdate && inputs.projectUpdate !== 'false') {
```

Comment:
`false` seems weird. Why not checking for empty string here?

Reply:
I mainly wanted to make `true` just run project update without args, otherwise, I would need to provide something, and that something would be provided to `dagger project update` as an argument. But that implied that 'false' would also have to be handled.

Can you elaborate on what you think are the behaviors expected for such a flag?

I tried to implement the following:
```yaml
project-update: 'false' # doesn't run dagger project update
project-update: 'true' # runs: `dagger project update`
project-update: 'package -flag' # runs: `dagger project update package -flag`
```

Reply:
> Can you elaborate on what you think are the behaviors expected for such a flag?

This is what I had in mind as a client: 

```yaml
project-update: "-flag" # runs with default package with -flag
project-update: "package -flag" # runs package with -flag
```

If you don't  specify the `project-update` field in the YAML, it's not used at all. 

WDYT?

Reply:
I tried to do that, but [getInput always return a string](https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L122) =/

And that also means that to run just `dagger project update` I would need to do like this
```yaml
project-update: ''
```

which I have mixed feelings about

Reply:
> I tried to do that, but [getInput always return a string](https://github.com/actions/toolkit/blob/main/packages/core/src/core.ts#L122) =/
> 
> And that also means that to run just `dagger project update` I would need to do like this
> 
> ```yaml
> project-update: ''
> ```
> 
> which I have mixed feelings about

but how would you run an empty `project update` with flags?

```yaml
project-update: 'true -flag' #?
```
This feels weird. 

Looking at the [`getInput` code](https://github.com/actions/toolkit/blob/27f76dfe1afb2b7e5e679cd8e97192d34d8320e6/packages/core/src/core.ts#L129-L130) I see that it's not actually checking if the variable actually is set or not. I just returns the default value, which is very strange. 

I'm not a very saavy actions user. Do you know how other actions treat these types of cases for `string` inputs that could not be set?

Reply:
> but how would you run an empty project update with flags?

```yaml
project-update: '-flag'
```

- if the flag is anything else other than `false` it will run `dagger project update`
- if the flag is only `true` it will just run `dagger project update`
- anything else will be appended to `dagger project update`

Reply:
It's @crazy-max :sweat_smile:

So I'm not sure I'm following but is `project-update` just a bool to run `dagger project update` or is it possible to also add flags?

Reply:
> It's @crazy-max

:woot: :D

> So I'm not sure I'm following but is `project-update` just a bool to run `dagger project update` or is it possible to also add flags?

`project-update` ideally should be a `string` argument since the current `dagger` command allows the user the specifc ya `package_name` as an argument and then additional flags.  So what we're trying to achieve here is a way where you can do: `dagger project update https://github.com/marcosnils/mypackage --some-flag`

Reply:
Ah sorry so there is an optional arg that define which package has to be updated. In this case user might also want to update multi package. So we should consider a multi-line input:

```yaml
      - name: dagger do test
        uses: dagger/dagger-for-github@v2
        with:
          project-update: |
            pkgfoo
            pkgbar
          args: do test
```

If nothing provided then `false`. If `true` runs `dagger project update`.

Reply:
So you could take the approach of https://github.com/docker/metadata-action#basic and define a pkg `key`:

```yaml
      - name: dagger do test
        uses: dagger/dagger-for-github@v2
        with:
          project-update: |
            pkg=foo
            pkg=bar
          args: do test
```

This way no collision possible.

Reply:
Ah didn't see your comment @marcosnils. So if there are also flags then a new key?:

```yaml
      - name: dagger do test
        uses: dagger/dagger-for-github@v2
        with:
          project-update: |
            pkg=foo,flags=--flag1;--flag2
            pkg=bar
          args: do test
```

A bit cryptic.. WDYT?

Reply:
> So if there are also flags then a new key?:

`dagger project update` currently  accepts only 1 package as an argument. If you want to update multiple, currently you have to run the command multiple times.

Reply:
Yes that's the purpose of the multi-line inputs. With https://github.com/dagger/dagger-for-github/pull/44#discussion_r847767313 it would run:

```
dagger project update foo --flag1 --flag2
dagger project update bar
```

Reply:
I talked about it in https://github.com/dagger/dagger-for-github/issues/19#issuecomment-1048631619

Reply:
> We could also not add a new input and reuse `args` as a multi-line input as I suggested in [#19 (comment)](https://github.com/dagger/dagger-for-github/issues/19#issuecomment-1048631619) so:

I like this approach.

Reply:
should I close this PR and open another to work on multiline args?

Reply:
@joaofnds Yes I think so. Before that I will open a PR to handle multi-line inputs I have already started to work on. After that you will be able to use this helper for the multi `args` input.

Reply:
It's not pretty, but I sort of got it working with a semicolon (:
https://github.com/joaofnds/ds/commit/ef62506a38060c3a4324e0d5902a789873d199d8

Reply:
Note that we should also deprecate `args` if we go this way as it would be confusing for the user. Instead we should name it `cmds` I think. Up to suggestion on this. cc @aluzzardi @marcosnils